lib/index.js

```diff
@@ -1,44 +1,28 @@
-module.exports = {
-  extends: [
-    'eslint:recommended',
-  ],
-  parserOptions: {
+const globals = require("globals")
+const js = require("@eslint/js")
+const pluginNode = require("eslint-plugin-node")
+const pluginImport = require("eslint-plugin-import")
```

Comment:
Does not yet support ESLint 9: https://github.com/import-js/eslint-plugin-import/issues/2948

See also: https://github.com/neostandard/neostandard/issues/15